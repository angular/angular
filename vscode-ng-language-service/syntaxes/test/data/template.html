<!-- Interpolation test -->
<div>{{ call(1 + 2 + 3) }}</div>