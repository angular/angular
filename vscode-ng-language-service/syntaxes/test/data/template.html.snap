><!-- Interpolation test -->
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^ template.ng
><div>{{ call(1 + 2 + 3) }}</div>
#^^^^^ template.ng
#     ^^ template.ng punctuation.definition.block.ts
#       ^ template.ng expression.ng
#        ^^^^ template.ng expression.ng entity.name.function.ts
#            ^ template.ng expression.ng meta.brace.round.ts
#             ^ template.ng expression.ng constant.numeric.decimal.ts
#              ^ template.ng expression.ng
#               ^ template.ng expression.ng keyword.operator.arithmetic.ts
#                ^ template.ng expression.ng
#                 ^ template.ng expression.ng constant.numeric.decimal.ts
#                  ^ template.ng expression.ng
#                   ^ template.ng expression.ng keyword.operator.arithmetic.ts
#                    ^ template.ng expression.ng
#                     ^ template.ng expression.ng constant.numeric.decimal.ts
#                      ^ template.ng expression.ng meta.brace.round.ts
#                       ^ template.ng expression.ng
#                        ^^ template.ng punctuation.definition.block.ts
#                          ^^^^^^^ template.ng